src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +34,10 @@ public List<Person> extractTeenagers(final List<Person> people) {
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+        Set<String> emailSet = new HashSet<>(emails);
```

Comment:
To achieve uniqueness you can use `.distinct()` stream operator instead.
Please, give it a try:)